### PR TITLE
[ANALYZER-3593] - Analyzer Multi-Chart is hard-coded to show only 50 charts

### PR DESCRIPTION
**Pull request by renato-s**
@andreramos89 
@bcostahitachivantara 
@smmribeiro

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 55s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: ccc-js/src/main/javascript/package-res/ccc/core/base/chart/chart.data.js [hunk 1]
```diff
@@ -672,6 +672,13 @@ pvc.BaseChart
             }
         }
 
+        if ( smallDatas.length > multiOption('Max') ) {
+            chart._lastRenderMessage = {
+                code: "maxChartsReached",
+                description: "The maximum multi-chart count has been reached. To see more charts increase the number of charts in the 'Chart Options'"
```
Comment (dcleao) [inline ccc-js/src/main/javascript/package-res/ccc/core/base/chart/chart.data.js:678]: "Chart Options" is an Analyzer thing, which CCC does not know about. Please, either remove that statement or replace it with one that mentions the `multiChartMax` CCC option.

**Review by dcleao — CHANGES_REQUESTED** [review]

File: ccc-js/src/main/javascript/package-res/ccc/core/base/chart/chart.data.js (hunk 1 above)
Comment (renato-s) [reply ccc-js/src/main/javascript/package-res/ccc/core/base/chart/chart.data.js]: Fixed

**Review by dcleao — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 50s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**